src/app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+const bodyParser = require('body-parser');
+
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname + '/..' + '/public/index.html'))
+})
+
+app.use(express.static('public'));
+
+app.listen(process.env.PORT || 5000, () => {
+  console.log('Echo listening on port 5000!')
```

Comment:
Above: `const port = process.env.PORT || 5000`
then
```js
app.listen(port, () => {
  console.log(`Echo listening on port ${port}!`)
}
```
I know in Hapi you can go something like: `server.info.port` not sure if express has equivalent but not important.